src/transformers/models/bort/convert_bort_original_gluonnlp_checkpoint_to_pytorch.py

```diff
@@ -0,0 +1,316 @@
+    # | -------------------------------------------------------------- | ----------------------
+    # | `encoder.layer_norm.beta`                                      | `bert.embeddings.LayerNorm.bias`
+    # | `encoder.layer_norm.gamma`                                     | `bert.embeddings.LayerNorm.weight`
+    # | `encoder.position_weight`                                      | `bert.embeddings.position_embeddings.weight`
+    # | `word_embed.0.weight`                                          | `bert.embeddings.word_embeddings.weight`
+    # | `encoder.transformer_cells.*.attention_cell.proj_key.bias`     | `bert.encoder.layer.*.attention.self.key.bias`
+    # | `encoder.transformer_cells.*.attention_cell.proj_key.weight`   | `bert.encoder.layer.*.attention.self.key.weight`
+    # | `encoder.transformer_cells.*.attention_cell.proj_query.bias`   | `bert.encoder.layer.*.attention.self.query.bias`
+    # | `encoder.transformer_cells.*.attention_cell.proj_query.weight` | `bert.encoder.layer.*.attention.self.query.weight`
+    # | `encoder.transformer_cells.*.attention_cell.proj_value.bias`   | `bert.encoder.layer.*.attention.self.value.bias`
+    # | `encoder.transformer_cells.*.attention_cell.proj_value.weight` | `bert.encoder.layer.*.attention.self.value.weight`
+    # | `encoder.transformer_cells.*.ffn.ffn_2.bias`                   | `bert.encoder.layer.*.attention.output.dense.bias`
+    # | `encoder.transformer_cells.*.ffn.ffn_2.weight`                 | `bert.encoder.layer.*.attention.output.dense.weight`
+    # | `encoder.transformer_cells.*.layer_norm.beta`                  | `bert.encoder.layer.*.attention.output.LayerNorm.bias`
+    # | `encoder.transformer_cells.*.layer_norm.gamma`                 | `bert.encoder.layer.*.attention.output.LayerNorm.weight`
+    # | `encoder.transformer_cells.*.ffn.ffn_1.bias`                   | `bert.encoder.layer.*.intermediate.dense.bias`
+    # | `encoder.transformer_cells.*.ffn.ffn_1.weight`                 | `bert.encoder.layer.*.intermediate.dense.weight`
+    # | `encoder.transformer_cells.*.ffn.layer_norm.beta`              | `bert.encoder.layer.*.output.LayerNorm.bias`
+    # | `encoder.transformer_cells.*.ffn.layer_norm.gamma`             | `bert.encoder.layer.*.output.LayerNorm.weight`
+    # | `encoder.transformer_cells.*.proj.bias`                        | `bert.encoder.layer.*.output.dense.bias`
+    # | `encoder.transformer_cells.*.proj.weight`                      | `bert.encoder.layer.*.output.dense.weight`
+
+    # Helper function to convert MXNET Arrays to PyTorch
+    def to_torch(mx_array) -> torch.nn.Parameter:
+        return torch.nn.Parameter(torch.FloatTensor(mx_array.data().asnumpy()))
+
+    # Check param shapes and map new HF param back
+    def check_and_map_params(hf_param, gluon_param):
+        shape_hf = hf_param.shape
+
+        gluon_param = to_torch(params[gluon_param])
+        shape_gluon = gluon_param.shape
+
+        assert shape_hf == shape_gluon
```

Comment:
maybe a small assert message would be great just:

```suggestion
        assert shape_hf == shape_gluon, f"The gluon parameter {gluon_param} has shape {shape_gluon}, but expects shape {shape_hf} for Transformers"
```